units/user@.service.in

```diff
@@ -10,7 +10,6 @@
 [Unit]
 Description=User Manager for UID %i
 Documentation=man:user@.service(5)
-After=systemd-user-sessions.service user-runtime-dir@%i.service dbus.service
```

Comment:
this latter two units should remain here, they are not really optional.

Reply:
You meant `user-runtime-dir@%i.service` and  `dbus.service` ? They're actually now specified in the drop-ins.

Or maybe I missed your point ?

Reply:
yeah, and i think they should remain in the main unit file.

Reply:
Ah ok, true!